app/containers/ItuensContainer/apiUtils.ts

```diff
@@ -0,0 +1,19 @@
+import { create } from 'apisauce';
+
+const api = create({ baseURL: 'https://itunes.apple.com' });
+
+const useApiSauce = () => {
+  const apiCall = async (param: any) => {
+    console.log(param);
+    try {
+      const res = await api.any({ method: 'GET', url: `/search?term=${param}` });
+      return res;
+    } catch (error) {
+      return error;
+    }
+  };
+
+  return apiCall;
+};
+
+export default useApiSauce;
```

Comment:
shouldn't this file be in the utils folder ? , also fix typo in filename .

Reply:
I have moved this file in service folder in feat/created-UI branch

Reply:
ok